compact/bhcal.xml

```diff
@@ -0,0 +1,14 @@
+<lccdd>
+
+  <display>
+  </display>
+  <define>
+  </define>
+
+  <documentation level="10">
+    ### Barrel Hcal configuration
+  </documentation>
+
+  <include ref="hcal/hcal_barrel_gdml.xml"/>
+
+</lccdd>
```

Comment:
We probably don't need this file. Most subsystems are labeled 'little endian': e.g. `ecal_barrel_sciglass.xml`. So here I'd probably just keep the name `hcal_barrel_gdml.xml` in line with that approach.

Reply:
I created this file to have a configuration that allowed me to debug the new barrel hcal development without interaction with all the other development going on. Are you suggesting it disappear, or it be renamed hcal_barrel_gdml.xml?

Reply:
Nevermind - I understood this question in the context of another question below